sdk-api-src/content/versionhelpers/nf-versionhelpers-iswindowsversionorgreater.md

```diff
@@ -1,7 +1,8 @@
 ---
 UID: NF:versionhelpers.IsWindowsVersionOrGreater
 title: IsWindowsVersionOrGreater function (versionhelpers.h)
-description: Indicates if the current OS version matches, or is greater than, the provided version information.
helpviewer_keywords: ["IsWindowsVersionOrGreater","IsWindowsVersionOrGreater function","base.iswindowsversionorgreater","versionhelpers/IsWindowsVersionOrGreater"]
+description: Indicates if the current OS version matches, or is greater than, the provided version information.
+helpviewer_keywords: ["IsWindowsVersionOrGreater","IsWindowsVersionOrGreater function","base.iswindowsversionorgreater","versionhelpers/IsWindowsVersionOrGreater"]

```

Comment:
Note there's an EOL change in there. I did the edit on the github website (using MsEdge), and that was auto-done for me (not sure why).

Reply:
Yep, the tools do that.